content/blog/2019/r-version-support/index.md

```diff
@@ -15,36 +15,24 @@ rmd_hash: efe6168c59ef5121
 
 ---
 
-## Which versions of R do tidyverse packages support?
+Our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R. This policy applies to all packages that we support, i.e. as well as all tidyverse packages, it also applies to the infrastructure packages that we maintain in [r-lib](https://github.com/r-lib).
 
-Though package-specific minimum versions of R are given in the `Depends` fields of individual `DESCRIPTION` files, our general policy is to support the **current version**, the **devel version**, and **four previous versions** of R.
+This, the *official* minimum supported versions of R for the tidyverse are as described here:
 
-**Edit 2022-11-15[^1]**
+| Date       | Released R version | Minimum supported version |
+|-----------:|:-------------------|:--------------------------|
+| 2019-04-26 | 3.6                | 3.2                       |
+| 2020-04-24 | 4.0                | 3.3                       |
+| 2021-05-18 | 4.1                | 3.4                       |
+| 2022-04-22 | 4.2                | 3.5                       |
+| 2023-04-21 | 4.3                | 3.6                       |
+| 2024-04-24 | 4.4                | 4.0                       |
+| 2025       | 4.5                | 4.1                       |
+| 2026       | 4.6                | 4.2                       |
 
-Over the past several years and moving forward (as described in the "Release plans" section of the [R Developer Page](https://developer.r-project.org/)), R version releases occur annually:
+Note that, as described in the "Release plans" section of the [R Developer Page](https://developer.r-project.org/), R version releases occur annually in Spring. We generally update the required version in package metadata on the next package release after the R package, so you may see older versions listed in published CRAN packages.
```

Comment:
This doesn't seem right:

"on the next package release after the R package"

After the R release?